clusterware-customize/libexec/customize/actions/help

```diff
@@ -152,6 +153,42 @@ help_for_trigger() {
 EOF
 }
 
+help_for_push() {
+  cat <<EOF
+  SYNOPSIS:
+
+    alces customize push
+
+  DESCRIPTION:
+
+    Push a customization profile to an upstream repository.
+EOF
+}
+
+help_for_slave() {
+  cat <<EOF
+  SYNOPSIS:
+
+    alces customize slave [OPTIONS]
+
+  DESCRIPTION:
+
+    Modifies the list of profiles to be ran when a compute node boots.
+    Has the option to add, remove or list the profiles.
+
+  OPTIONS:
```

Comment:
I have gone with the latter option, however I have laid some of the ground work if we want to switch `customize slave` to match `job-queue`